code/obj/critter/drone.dm

```diff
@@ -5,7 +5,7 @@
 	icon_state = "drone"
 	density = 1
 	health = 65
-	var/maxhealth = 65 // for damage description
+	maxhealth = 65 // for damage description
```

Comment:
this seems a little off-task, but also wouldn't it be better off as var/num/maxhealth?